share/spice/hex/hex.js

```diff
@@ -84,4 +80,8 @@
             });
         });
     };
-}(this));
\ No newline at end of file
+
+    Handlebars.registerHelper('hex_plural', function(number, options) {
```

Comment:
We actually have a helper for this :)

https://github.com/duckduckgo/duckduckgo-template-helpers/blob/master/template_helpers.js#L666